indexd/index/version_data.py

```diff
@@ -1,2 +1,2 @@
-VERSION = ""
-COMMIT = ""
+COMMIT = "c61ead7c4ca94104b8a8988edf4e5598a84757d8"
+VERSION = "2.11.0-51-gc61ead7"
```

Comment:
What are these versions? Do we need to do something special to set them next time?

Reply:
This is updated each time we run the docker.sh file. Tried to keep it as is for now, but we may have to come back to it